src/App.tsx

```diff
@@ -2,12 +2,29 @@ import { useState } from 'react';
 import './App.css';
 import { Game } from './components/Game/Game';
 import { SelectPokemons } from './components/SelectPokemons/SelectPokemons';
+import { PokemonsData } from './types/PokemonsData';
 
 export const App = (): JSX.Element => {
-  const disabledPokemons = [''];
-  const selectedPokemonSize = 1;
   const [isSelected, setIsSelected] = useState(false);
   const [isGameStarted, setIsGameStarted] = useState(false);
+  const [pokemonPlayer, setPokemonPlayer] = useState<PokemonsData>();
+  const [selectedPokemons, setSelectedPokemons] = useState<string[]>([]);
+
+  const onSelectPokemon = (pokemon: PokemonsData, isCurrentlySelected: boolean): void => {
+    const disabledPokemons = [''];
+    const selectedPokemonSize = 1;
+    setPokemonPlayer(pokemon);
+    if (isCurrentlySelected) {
+      setIsSelected((prevState) => !prevState);
```

Comment:
Updater here is unneeded.
```suggestion
      setIsSelected(!isSelected);
```